src/Avalonia.Controls/HotkeyManager.cs

```diff
@@ -85,6 +88,8 @@ void Register()
                 if (_root != null && _hotkey != null)
                 {
                     _binding = new KeyBinding() {Gesture = _hotkey, Command = _wrapper};
+                    _commandParameterChangedSubscriber = _binding.Bind(KeyBinding.CommandParameterProperty
+                        , ExpressionObserver.Create(((ICommandSource)_control), o => o.CommandParameter));
```

Comment:
Nit: we usually put commas on the previous line.

```suggestion
                    _commandParameterChangedSubscriber = _binding.Bind(KeyBinding.CommandParameterProperty,
                        ExpressionObserver.Create(((ICommandSource)_control), o => o.CommandParameter));
```

Reply:
Actually, is a binding really needed here? The `HotkeyCommandWrapper` class has access to `ICommandSource.CommandParameter` - why not just use that?

Reply:
You are right.